mocks/publications/galleries/38.json

```diff
@@ -6,7 +6,7 @@
         "documentId": 38,
         "contentType": "gallery",
         "documentType": "article",
-        "layout": "gallery",
+        "layout": "asset",
```

Comment:
I think `layout` and `contentType` need to be the same string. Correct @peyerluk ?

Reply:
We shouldn't need layout anymore. We can just delete that.

Reply:
I have removed the asset-layout component and the "layout": "asset" definitions.